awscli/testutils.py

```diff
@@ -181,6 +181,45 @@ def create_bucket(session, name=None, region=None):
     return bucket_name
 
 
+def create_dir_bucket(session, name=None, location=None):
+    """
+    Creates a S3 directory bucket
+    :returns: the name of the bucket created
+    """
+    if not location:
+        location = ('us-west-2', 'usw2-az1')
+    region, az = location
```

Comment:
nit: name the parameter `availability_zone` instead of `az` to avoid using abbreviations.